telco-examples/mgmt-cluster/airgap/README.md

```diff
@@ -37,7 +37,20 @@ You need to modify the `${MGMT_CLUSTER_IP}` with the Node IP in the following fi
 
 You need to modify the following folder:
 
-- `base-images` - To include inside the `SL-Micro.x86_64-6.0-Default-SelfInstall-GM2.install.iso` image downloaded from the SUSE Customer Center.
+- `base-images` - To include the raw image generated by Kiwi as:
+
+```
+mkdir output
+sudo podman run --privileged -v /etc/zypp/repos.d:/micro-sdk/repos/ -v $(pwd)/output:/tmp/output -it registry.opensuse.org/isv/suse/edge/factory/images_16.0/kiwi-builder:10.2.12.0 build-image -p Default-SelfInstall
+```
```

Comment:
This should use the product image e.g `registry.suse.com/edge/3.3/kiwi-builder:10.2.12.0`

Reply:
changed! and all references